cardano-node/src/Cardano/Tracing/Tracers.hs

```diff
@@ -541,33 +544,38 @@ teeForge' tr =
     meta <- mkLOMeta Critical Confidential
     traceNamedObject (appendName "metrics" tr) . (meta,) $
       case ev of
-        Consensus.TraceNodeCannotForge slot _reason ->
-          LogValue "nodeCannotForge" $ PureI $ fromIntegral $ unSlotNo slot
-        Consensus.TraceForgeStateUpdateError slot _reason ->
-          LogValue "forgeStateUpdateError" $ PureI $ fromIntegral $ unSlotNo slot
-        Consensus.TraceForgedBlock slot _ _ _ ->
-          LogValue "forgedSlotLast" $ PureI $ fromIntegral $ unSlotNo slot
         Consensus.TraceStartLeadershipCheck slot ->
           LogValue "aboutToLeadSlotLast" $ PureI $ fromIntegral $ unSlotNo slot
+        Consensus.TraceSlotIsImmutable slot _tipPoint _tipBlkNo ->
+          LogValue "slotIsImmutable" $ PureI $ fromIntegral $ unSlotNo slot
+        Consensus.TraceBlockFromFuture slot _slotNo ->
+          LogValue "blockFromFuture" $ PureI $ fromIntegral $ unSlotNo slot
+        Consensus.TraceBlockContext slot _tipBlkNo _tipPoint ->
+          LogValue "blockContext" $ PureI $ fromIntegral $ unSlotNo slot
```

Comment:
@deepfire What about this one (and `"ledgerState"`)? As you know, they're only for debug purposes. I don't know how these `LogValue`s are used, so up to you.

Reply:
I think this should just remain configurable -- if it's not traced then it's configuration-by-editing at debug time.